content/doc/developer/plugin-development/incrementals.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: developersection
+title: "Developing plugins and Jenkins core parallely"
+tags:
+- jenkins
+- incrementals
+- core
+- plugin
+description: "This article explains how to use Incrementals for developing plugins parallely with Jenkins core"
+---
+
+A common problem one might stumble upon is developing an API in Jenkins core and simultaneously
+building a reference implementation inside a plugin.
+What to do, one might ask.
+
+Worry not, Incrementals are here to the rescue!
+
+With incrementals, we can create Jenkins core incremental versions which can be used along with the plugin to build the
+plugin against the very latest changes you have made to core.
+Also, it allows keeping track and switching between incremental versions.
+Not only that, it allows others to also not have to build Jenkins locally to try out the changes or collaborate with
+you on your awesome project!
+
+Sounds cool? Let's see how to go about it.
+
+== 1. Create a Pull Request in Jenkins core
+
+First step is to create a PR on github link:https://github.com/jenkinsci/jenkins/pulls[here] by clicking the "New Pull
```

Comment:
please don't use the term 'here' in links it's not as clear as it could be, see https://uxmovement.com/content/why-your-links-should-never-say-click-here/, it's also not very accessible content

A link should describe where it's going to be the text of the page, i.e. be more descriptive than just writing here

possibly this would be a good option:
```suggestion
First step is to create a PR on link:https://github.com/jenkinsci/jenkins/pulls[GitHub] by clicking the "New Pull
```

Reply:
Thanks Tim for the article, nice read :) I have made the requested changes